core/vm/common.go

```diff
@@ -28,6 +28,8 @@ import (
 // Global Debug flag indicating Debug VM (full logging)
 var Debug bool
 
+var GenerateStructLogs bool = false
```

Comment:
Though I'd say for this PR it's fine. I'll look in to this with https://github.com/ethereum/go-ethereum/pull/2141